tests/e2e/api/controllers/users.js

```diff
@@ -950,19 +950,24 @@ describe('Users API', () => {
         await utils.updateSettings(settings, true);
         await utils.addTranslations('en', { token_login_sms: 'Instructions sms' });
 
-        try {
-          await utils.request({ path: '/api/v1/users', method: 'POST', body: users });
-          chai.assert.fail('Should have thrown');
-        } catch (error) {
-          const response = error.responseBody;
-          chai.expect(response.code).to.equal(400);
-          chai.expect(response.error).include('Missing fields password for user at index 1');
-          chai.expect(response.details).to.deep.equal({
-            failingIndexes: [
-              { fields: ['password'], index: 1 },
-            ],
-          });
-        }
+        const response = await utils.request({ path: '/api/v1/users', method: 'POST', body: users });
```

Comment:
```suggestion
        const response = await utils.request({ path: '/api/v2/users', method: 'POST', body: users });
```

I wonder if we should keep the test for v1 and then also test in v2

Reply:
Decided to keep it as it is.